pom.xml

```diff
@@ -0,0 +1,475 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.exasol</groupId>
+    <artifactId>small-json-files-test-fixture</artifactId>
+    <version>0.1.0</version>
+    <name>Small Json Files Test Fixture</name>
+    <description>Test fixture with many small JSON files on S3.</description>
+    <url>https://github.com/exasol/small-json-files-test-fixture</url>
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <java.version>11</java.version>
+        <junit.version>5.8.1</junit.version>
+        <jacoco.version>0.8.7</jacoco.version>
+        <gpg.skip>true</gpg.skip>
+        <!--
+            M4 does not fail tests if beforeAll fails
+            M5 fails in integration tests with a strange error
+            -->
+        <surefire.and.failsafe.plugin.version>3.0.0-M3</surefire.and.failsafe.plugin.version>
```

Comment:
Is this fixed with the latest version?

Reply:
I don't think so. @redcatbear tried out some new milestones and ran into different issues.